### ISSUE TITLE
JAVA

### ISSUE THREAD

**Issue by tobiasko** [Nesvilab/FragPipe#131]
# Describe the problem

* **I'm submitting a:**
  - [x] problem running the software
  - [ ] bug report
  - [ ] feature request
  - [ ] question

## System info

Windows 10 Enterprise 2016 LTSB

## Problem

I installed Prebuilt OpenJDK Binaries (AdoptOpenJDK JDK with Hotspot 8.0.222.10 (x64) on my system, but FragPipe still reports:

"The application requires a JAVA runtime environment 1.8.0"

and forwards me to https://adoptopenjdk.net/

![image](https://user-images.githubusercontent.com/12233339/64431519-e706fc80-d0ba-11e9-8b65-353f04545786.png)

What do I need?

**Comment by chhh** [comment]
That should be enough to run. Can you start a console and see what `java -version` says?
have you tried restarting your computer or just log off and then log on for the changes to environment variables to take effect?

Did you download the `.exe` or `.zip`? If `.exe`, then try the zip - there's `bin` directory with `FragPipe.bat` file to start the application.

**Comment by tobiasko** [comment]
C:\Users\Admin>java -version
openjdk version "1.8.0_222"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_222-b10)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.222-b10, mixed mode)

**Comment by tobiasko** [comment]
C:\Users\Admin>java -version
openjdk version "1.8.0_222"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_222-b10)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.222-b10, mixed mode)

**Comment by tobiasko** [comment]
I tried the ```FragPipe.bat``` that gave the following messages on the console:

```
12:37:22.147 [AWT-EventQueue-0] ERROR u.msfragger.gui.MsfraggerGuiFrame - Could not read fragpipe form cache from: C:\Users\Admin\AppData\Local\Temp\fragpipe\fragpipe-forms.cache
12:37:22.301 [pool-1-thread-1] ERROR u.m.p.f.MsfraggerVersionComparator - v1 was determined to be both old and new scheme, something is wrong
12:37:22.302 [pool-1-thread-1] ERROR u.m.p.f.MsfraggerVersionComparator - v1 was determined to be both old and new scheme, something is wrong
12:37:22.611 [pool-1-thread-1] ERROR u.m.p.f.MsfraggerVersionComparator - v1 was determined to be both old and new scheme, something is wrong
12:37:22.948 [pool-1-thread-1] ERROR u.m.p.f.MsfraggerVersionComparator - v1 was determined to be both old and new scheme, something is wrong
12:37:23.176 [pool-1-thread-1] ERROR u.m.p.f.MsfraggerVersionComparator - v1 was determined to be both old and new scheme, something is wrong
12:37:23.177 [pool-1-thread-1] ERROR u.m.p.f.MsfraggerVersionComparator - v1 was determined to be both old and new scheme, something is wrong
```
But the GUI still started up.

**Comment by guoci** [comment]
@tobiasko Those error messages has been corrected recently. You can try the new version below:
https://drive.google.com/file/d/1qKnEEf84X7HBAOXq2yiZ8VL_Bz3mXsB7/view

**Comment by chhh** [comment]
@tobiasko This has been fixed in version 10.1: https://github.com/Nesvilab/FragPipe/releases/tag/10.1
